### PR TITLE
JBPM-6784 Jobs: New job dialog "Add Parameter" button doesn't respond…

### PR DESCRIPTION
**Pull request by bxf12315**
… to click

@cristianonicolai  @bsremac 
It's work-around way, Please review it.

### CODE REVIEW

**Comment by bsremac** [comment]
@bxf12315 Can you please squash all the commits into one? I can't pull all the changes from the PR like this.

**Comment by jhrcek** [comment]
@bsremac +1 about squashing, but you should be able to pull any PR using `git fetch upstream pull/$PRNUMBER/head:NameOfLocalPRBranch$PRNUMBER` (and delete branch if you have it from the past `git branch -D $BRANCH_NAME`

**Comment by bxf12315** [comment]
@bsremac  NP, I just want to keep PR history , if it is ok, I will rebase all commit .

**Comment by bxf12315** [comment]
@bsremac done

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java [hunk 1]
```diff
@@ -201,7 +204,15 @@ public void init() {
         initGridColumns();
         initDateTimePicker();
 
+        Button button = new Button(Constants.INSTANCE.Add_Parameter(), new ClickHandler() {
+            @Override
+            public void onClick(ClickEvent event) {
+                addNewParameter();
+            }
+        });
+        button.addStyleName("btn btn-default");
```
Comment (cristianonicolai) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java]: Setting the style isn't needed for this component as this is the default already.

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java [hunk 2]
```diff
@@ -201,7 +204,15 @@ public void init() {
         initGridColumns();
         initDateTimePicker();
 
+        Button button = new Button(Constants.INSTANCE.Add_Parameter(), new ClickHandler() {
```
Comment (cristianonicolai) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java]: Please use _GWT.create_ instead, as this is needed for GWTMockito to work properly.
I would suggest something like:
`Button button = GWT.create(Button.class);`
 `button.setText(Constants.INSTANCE.Add_Parameter());`
`button.addClickHandler(e -> addNewParameter());`

Comment (bxf12315) [reply jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java]: Ok

**Comment by bxf12315** [comment]
@cristianonicolai  Updated

**Comment by bsremac** [comment]
Depends on https://github.com/jboss-integration/bxms-qe-tests/pull/1554

**Comment by cristianonicolai** [comment]
@bxf12315 thanks

**Pull request merged**